### PR TITLE
Handling events

### PR DESCRIPTION
**Pull request by Shakir-Hussain12**
**For this PR, I:**
- Initialized my app with `Vite` instead of `CRA`.
- Configured and initialized Linters.
- Followed the links to the code to create a todo-list app.
- Followed GitFlow practices to maintain the project.
- Divided the project into components for React best practices.
- Used React states, hooks, and handlers.
- Used local storage to store data locally.
- Added Styling to the project.

### CODE REVIEW

File: src/components/TodosList.jsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+/* eslint-disable react/prop-types */
+import TodoItem from './InputTodo';
+
```
Comment (serengia) [inline src/components/TodosList.jsx]: Please import the right component to be able to correctly display your todo items.

**Review by serengia — CHANGES_REQUESTED** [review]
> # Changes Required 
> ### Hi @Shakir-Hussain12 
> Great job so far, you are almost there.
> 
> 
> 
> ## (Highlights) Good Points: 
> * Good commit messages. :heavy_check_mark:
> * No linter errors. :heavy_check_mark:
> * Followed Github flow :heavy_check_mark:
> * PR has a good title and summary :heavy_check_mark:
>
> ## (Changes Required) Aspects to improve: 
> * [x]  As shown in the screenshot below, you seem to correctly store todos in local storage. However, your todos are not being properly displayed.
> * [x]  Please check my inline comments.
> * [x]  Also update the readme file to reflect the project.

> 
> Screenshot (6)

> 
> ## [Optional] suggestions:
> * Nothing to mention.
> 
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> **Cheers and Happy coding! :clap::clap::clap:**
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear
> Please, remember to tag @serengia in your question so I can receive the notification.
>
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/components/TodosList.jsx (hunk 1 above)
Comment (Shakir-Hussain12) [reply src/components/TodosList.jsx]: Thanks @serengia, I will make sure to add these changes ASAP.

**Review by serengia — APPROVED** [review]
### Hi @Shakir-Hussain12 , 

# Status: Approved ✔️
Excellent job on making the changes requested changes. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
>
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Followed Github flow :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and description :heavy_check_mark:
 
> ## [Optional] suggestions:
> * [ ]  You can center the todo list on the page. That way, you will improve the overall look of your project.
> 
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @serengia  in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**